azurerm/resource_arm_healthcare_service.go

```diff
@@ -0,0 +1,436 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/healthcareapis/mgmt/2018-08-20-preview/healthcareapis"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/tf"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/validate"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmHealthcareService() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmHealthcareServiceCreateUpdate,
+		Read:   resourceArmHealthcareServiceRead,
+		Update: resourceArmHealthcareServiceCreateUpdate,
+		Delete: resourceArmHealthcareServiceDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.NoEmptyStrings,
+			},
+
+			"location": azure.SchemaLocation(),
+
+			"resource_group_name": azure.SchemaResourceGroupName(),
+
+			"kind": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "fhir",
+				ValidateFunc: validation.StringInSlice([]string{
+					"fhir",
+					"fhir-Stu3",
+					"fhir-R4",
```

Comment:
Could we use the constants from the SDK here?

Reply:
done